database/migrations/2018_01_19_161213_create_attendances_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAttendancesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('attendances', function (Blueprint $table) {
+            $table->bigInteger('id');
+            $table->integer('school_id')->unsigned();
+            $table->integer('class_group_id')->unsigned()->nullable();
+            $table->integer('stream_id')->unsigned()->nullable();
+            $table->integer('teacher_id')->unsigned();
+            $table->integer('student_id')->unsigned();
+            $table->enum('gender', ['Male', 'Female', 'Other']);
+            $table->date('date');
+            $table->boolean('is_present');
+            $table->timestamps();
+        });
+
+        Schema::table('attendances', function($table) {
+            // $table->foreign('school_id')->refrences('id')->on('schools');
+            // $table->foreign('class_group_id')->refrences('id')->on('class_groups');
+            // $table->foreign('stream_id')->refrences('id')->on('streams');
+            // $table->foreign('teacher_id')->refrences('id')->on('teachers');
+            // $table->foreign('student_id')->refrences('id')->on('students');
```

Comment:
dude, it's a typo :laughing: :smile_cat: `references`

Reply:
:rofl: :rofl: :rofl: :rofl: